app/models/tag.rb

```diff
@@ -0,0 +1,12 @@
+class Tag < ApplicationRecord
+  include Insertable
```

Comment:
why included in tag 
should be included in base class.

Reply:
if I was still using it, it should've been included in `ApplicationRecord`?